lnt/tests/test_suite.py

```diff
@@ -449,6 +446,8 @@ def _configure(self, path, extra_cmake_defs=[], execute=True):
             defs['CMAKE_C_COMPILER'] = self.opts.cc
         if self.opts.cxx:
             defs['CMAKE_CXX_COMPILER'] = self.opts.cxx
+        if self.opts.make is not None:
```

Comment:
why not
```
if self.opts.make:
```
but also the default is `make` right, so this will always be set? `default="make"`, Do you need to remove the default value?

Reply:
Woops, the default should be removed now. Otherwise overriding the generator won't work without also specifying --use-make